takhojin/app.js

```diff
@@ -55,6 +47,62 @@ app.post("/users", async (req, res, next) => {
   res.status(201).json({ mesasage: " userCreated " });
 });
 
+app.post("/posts", async (req, res, next) => {
+  const { title, description, image, user_id } = req.body;
```

Comment:
```suggestion
  const { title, description, image, userId } = req.body;
```
자바스크립트에서는 언더바를 사용하는 camel_case는 쓰지 않는 것이 컨벤션 입니다. camelCase를 사용해서 코드가 작성될 수 있게 해주세요.